mod/schema/Unit.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
Slightly different from UnitCard.json
Does not accept formatting with "content": "foobar.json"